### PR TITLE
Add letters intro page

### PR DESCRIPTION
**Pull request by egarreau**
screen shot 2018-03-23 at 4 47 28 pm

### CODE REVIEW

File: script/build.js [hunk 1]
```diff
@@ -451,7 +451,7 @@ if (options.watch) {
         { from: '^/health-care/health-records(.*)', to: '/health-care/health-records/' },
         { from: '^/health-care/messaging(.*)', to: '/health-care/messaging/' },
         { from: '^/health-care/prescriptions(.*)', to: '/health-care/prescriptions/' },
-        { from: '^/letters(.*)', to: '/letters/' },
+        { from: '^/download-va-letters(.*)', to: '/download-va-letters/letters/' },
```
Comment (cehsu) [inline script/build.js]: We will want to add `/letters` to the `from`, I think.
{ from: '^/download-va-letters/letters(.*)', to: '/download-va-letters/letters/' }

File: src/js/user-profile/components/AuthApplicationSection.jsx [hunk 2]
```diff
@@ -16,6 +16,7 @@ class AuthApplicationSection extends React.Component {
             <p key="claims"><a href="/track-claims">Check your claim and appeal status</a></p>
             <p key="post-911"><a href="/education/gi-bill/post-9-11/ch-33-benefit">Get your Post-9/11 GI Bill statement of benefits</a></p>
             <p key="vic"><a href="/veteran-id-card">Apply for a Veteran ID Card</a></p>
+            <p key="letters"><a href="/download-va-letters">Download VA Letters</a></p>
```
Comment (cehsu) [inline src/js/user-profile/components/AuthApplicationSection.jsx]: Same here as above, I think.

**Review by cehsu — COMMENTED** [review]
Looks good so far. I had a few questions regarding where we plan to route users.

File: content/pages/download-va-letters.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
+---
+title: Download VA Letters
+layout: page-breadcrumbs.html
```
Comment (cehsu) [inline content/pages/download-va-letters.md]: Let's add `template: level2-index` to the metadata to make the formatting consistent.

File: content/pages/download-va-letters.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+---
+<script src="https://standards.usa.gov/assets/js/vendor/uswds.min.js" type="text/javascript"></script>
+
+<div class="main">
+<div class="section">
+<div class="row" markdown="1">
+
+# Download Your VA Letters
```
Comment (cehsu) [inline content/pages/download-va-letters.md]: We should be able to remove this if we update the metadata as mentioned above.
Also, should the title be "Download VA Letters" or "Download Your VA letters"? If the latter, should the links in the nav and footer also be updated to match?

File: content/pages/download-va-letters.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
+---
+title: Download VA Letters
+layout: page-breadcrumbs.html
+---
+<script src="https://standards.usa.gov/assets/js/vendor/uswds.min.js" type="text/javascript"></script>
```
Comment (cehsu) [inline content/pages/download-va-letters.md]: Let's move this to the bottom of the file.

File: content/pages/download-va-letters.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+<div class="main">
+<div class="section">
```
Comment (cehsu) [inline content/pages/download-va-letters.md]: According to the guidance provided in `navigation-links-list.html` we may want wrap each part with a heading into its own section, and replace the `<div class="section">` here with `<section>` and `<div class="main">` with `<main>`.

File: content/pages/download-va-letters.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+
+[View and print your Post-9/11 GI Bill statement of benefits.](/education/gi-bill/post-9-11/ch-33-benefit)
+
+[Sign in to eBenefits to request a Certificate of Eligibility for home loan benefits](https://eauth.va.gov/ebenefits/coe).
+
+[Sign in to eBenefits to request a copy of your discharge or separation papers (DD214)](https://eauth.va.gov/ebenefits/DPRIS).
+
+### What if I have trouble downloading a VA letter?
+
+Call the Vets.gov Help Desk at 1-855-574-7286 (TTY: 1-800-877-8339).
```
Comment (cehsu) [inline content/pages/download-va-letters.md]: @bethpotts is there any interest in making this phone number a clickable link?

File: content/pages/download-va-letters.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+
+<div class='va-introtext'>
+To receive some benefits, Veterans need a letter proving their status. You can access and download some of these benefit letters and documents online.
+</div>
+
+### What types of VA letters can I download using this tool?
+
+You can download your VA letters for benefits you’re currently receiving. You can also download letters for benefits you’re eligible for based on your service history.
+
+<a href='/download-va-letters/letters'><button class='usa-button-primary'>Access Your VA Letters</button></a>
```
Comment (cehsu) [inline content/pages/download-va-letters.md]: We can remove the button here and use it's class to style the anchor to look the same:
```
<a href='/download-va-letters/letters' class='usa-button-primary'>Access Your VA Letter</a>

```

File: content/pages/download-va-letters.md [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+[Sign in to eBenefits to request a copy of your discharge or separation papers (DD214)](https://eauth.va.gov/ebenefits/DPRIS).
+
+### What if I have trouble downloading a VA letter?
+
+Call the Vets.gov Help Desk at 1-855-574-7286 (TTY: 1-800-877-8339).
+We’re here Monday through Friday, 8:00 a.m. to 8:00 p.m. (ET).
+
+</div>
+</div>
+</div>
```
Comment (cehsu) [inline content/pages/download-va-letters.md]: It looks like we may want to add some `<br>` at the end of the file here to make the spacing consistent with the `/families-caregivers` page.

File: content/pages/download-va-letters.md [hunk 10]
```diff
@@ -0,0 +1,104 @@
+---
```
[the remark this answers is not in the document]
Comment (egarreau) [reply content/pages/download-va-letters.md]: Will this make it part of the app? We don't want it to be login gated.

[the remark this answers is not in the document]
Comment (cehsu) [reply content/pages/download-va-letters.md]: Nope it won't change the routing, it just allows us to keep the pages folder tidier.

File: content/pages/download-va-letters.md (hunk 3 above)
Comment (cehsu) [reply content/pages/download-va-letters.md]: @cvalarida would adding `permalink` here make sense as well?

File: src/js/user-profile/components/AuthApplicationSection.jsx (hunk 2 above)
Comment (egarreau) [reply src/js/user-profile/components/AuthApplicationSection.jsx]: The link URL you mean? It should go to the intro page (`/download-va-letters`) not the app (`/download-va-letters/letters`)

File: content/pages/download-va-letters.md (hunk 4 above)
Comment (cehsu) [reply content/pages/download-va-letters.md]: @bethpotts ^

File: src/js/user-profile/components/AuthApplicationSection.jsx (hunk 2 above)
Comment (cehsu) [reply src/js/user-profile/components/AuthApplicationSection.jsx]: Thanks for the helpful context, @egarreau.

File: content/pages/download-va-letters.md (hunk 3 above)
Comment (egarreau) [reply content/pages/download-va-letters.md]: The page title has changed to `Download VA Benefit Letters and Documents` but the header is supposed to stay as `Download Your VA Letters`. Is there a way to use `template: level2-index` without having it pull the page title into the header? Or should I leave it out?

Comment (cehsu) [reply content/pages/download-va-letters.md]: Good question, looping in @cvalarida, should we add a `template_title` variable to the metadata to supply this information, which could then be used by the `level2-index` template to replace the default `title`, if provided?
Something like:
```
<h1>{{ template_title or title }}</h1>
```

File: content/includes/veteran-navigation.html [hunk 11]
```diff
@@ -267,6 +267,7 @@
               <!--<li><a class="login-required" href="/veteran-id-card/how-to-get" onClick="reportHeaderNav('Request Your Records and ID');">Apply for a Veteran ID Card</a></li>-->
               <li><a href="/discharge-upgrade-instructions/" onClick="reportHeaderNav('Request Your Records and ID->Apply For Discharge Upgrade');">How to Apply for a Discharge Upgrade</a></li>
               <li><a href="/health-care/health-records/" onClick="reportHeaderNav('Request Your Records and ID->Health Care');">Get Your VA Health Records</a></li>
+              <li><a href="/download-va-letters/" onClick="reportHeaderNav('Request Your Records and ID->Health Care');">Download VA letters</a></li>
```
Comment (cehsu) [inline content/includes/veteran-navigation.html]: It looks like we need to update this content as well, not sure what we should replace it with, perhaps "Download Your VA Letters"?

File: content/pages/download-va-letters.md (hunk 3 above)
Comment (egarreau) [reply content/pages/download-va-letters.md]: Never mind, I've been informed the change applies to both the page title and the header :)

File: script/build.js (hunk 1 above)
Comment (egarreau) [reply script/build.js]: Actually this causes `Cannot GET` errors.

File: content/includes/veteran-navigation.html (hunk 11 above)
Comment (cehsu) [reply content/includes/veteran-navigation.html]: Perhaps  “Request Your Records and ID->Download Your VA Letters” is more consistent?

File: content/pages/download-va-letters/index.md [hunk 12]
```diff
@@ -0,0 +1,119 @@
...
+[Sign in to eBenefits to request a Certificate of Eligibility for home loan benefits](https://eauth.va.gov/ebenefits/coe).
+
+[Sign in to eBenefits to request a copy of your discharge or separation papers (DD214)](https://eauth.va.gov/ebenefits/DPRIS).
+
+</section>
+<section>
+
+### What if I have trouble downloading a VA letter?
+
+Call the Vets.gov Help Desk at 1-855-574-7286 (TTY: 1-800-877-8339).
```
Comment (cehsu) [inline content/pages/download-va-letters/index.md]: @peggygannon @bethpotts do we want to provide a hyperlink for this phone number, we do so elsewhere, such as on the `/health-care/apply/` page.

File: content/pages/download-va-letters/index.md [hunk 13]
```diff
@@ -0,0 +1,119 @@
+---
+title: Download VA Letters and Documents
+layout: page-breadcrumbs.html
+template: level2-index
+---
+
+<main>
```
Comment (cehsu) [inline content/pages/download-va-letters/index.md]: It turns out `<main>` is [not IE11-friendly](https://caniuse.com/#feat=html5semantic). In that case, let's stick with `<div id="main">`, sorry to ask you to revert this. Also, it seems like the `<section>` tag immediately within that is not necessary, so let's go ahead and remove that.

File: content/pages/download-va-letters/index.md [hunk 14]
```diff
@@ -0,0 +1,119 @@
...
+<section>
+
+### What if I have trouble downloading a VA letter?
+
+Call the Vets.gov Help Desk at 1-855-574-7286 (TTY: 1-800-877-8339).
+We’re here Monday through Friday, 8:00 a.m. to 8:00 p.m. (ET).
+
+<br>
+<br>
+<br>
```
Comment (cehsu) [inline content/pages/download-va-letters/index.md]: We tend to place such breaks after the rest of the content.

**Review by cehsu — APPROVED** [review]
Looks good to me once the specs pass.

File: content/pages/download-va-letters/index.md (hunk 12 above)
Comment (peggygannon) [reply content/pages/download-va-letters/index.md]: @cehsu Yes, that would be good. Thanks for catching this.

**Comment by peggygannon** [comment]
@egarreau Yes, we're good as far as content goes, but I thought we still needed stakeholder review/feedback before pushing this page live? 

cc @djmunoz1

File: content/pages/download-va-letters/index.md (hunk 12 above)
Comment (cehsu) [reply content/pages/download-va-letters/index.md]: Thanks @peggygannon, changed.

**Pull request merged**